docs/2.9.0/docs/canton/usermanual/performance.rst

```diff
@@ -506,74 +506,111 @@ If you submit a ledger API command (which can have multiple Daml commands), then
-batch operation per informee group. You can use choice observers to align informees if necessary. 
-
-For example, if the below `doUpdate` is only visible to the `owner` party, then whether the `efficient` flag is set or not 
-will have a huge impact on the number of views created.
+Strategy for Protocol Version 6
++++++++++++++++++++++++++++++++
+
+Starting from protocol version 6, Canton creates a view for every action node in the transaction tree if the
+**participants that host their informees are not a subset of their parent view's informee participants**.
+A view's informee participants are all the participants that must be informed about the view.
+Therefore, to reduce the number of views, you should try to reduce either the number of times the set of informees grows
+or make sure most of your actions share the same pool of participants that must be informed. Alternatively, you can
+add informees to the action nodes close to the root (e.g., by including choice observers) so that their children nodes
+are aggregated in the initial view, since these children nodes will only target a subset of those participants.
+
+One concrete way to reduce the number of views being generated by a daml model is to write batch commands and group the
+operations on the ledger based on their informees' participants and create a single choice and batch operations per
+informee's participants group.
+
+You can also reduce the number of views being generated for the children of a node (i.e., parent-child views) when
+creating your daml model.
+With the bellow example as reference, instead of sending a batch with 20 exercise commands `Foo`,
+you can group your contracts by a set of stakeholders that cover the exercise's informee participants and send
+one exercise command on a generator contract with the list of contracts that in turn exercises the Foo choice
+per stakeholder group. This way you can perform these operations in a single view, instead of having them spawn multiple
+different views.
+
+For example, if the `doUpdate` is called from a choice visible to the `owner` party alone, then whether the `efficient`
+flag is set or not will have a huge impact on the number of views created.
 
 .. code:: daml
 
    doUpdate : Bool -> Party -> Party -> [ContractId Example] -> Update ()
-   doUpdate efficient owner obs batch = do 
-   if efficient then do 
-      -- Would be even more efficient if we can reuse the BatchFoo contract
-      batcher <- create BatchFoo with owner 
-      batcher `exercise` Run with batch = batch; obs = obs    
-   else do
-      -- Canton will create one view per exercise
-      forA_ batch (`exercise` Foo)    
-   pure ()
+   doUpdate efficient owner obs batch = do
+      if efficient then do
+         -- Would be even more efficient if we can reuse the BatchFoo contract
+         batcher <- create BatchFoo with owner
```

Comment:
Why "more efficient"? In PV 6, the create node should end up in the same view as the calling choice calling `doUpdate` (because the owner is the only stakeholder and therefore present in the parent node). So the create doesn't change the number of views.

Reply:
Makes sense. I'll remove this comment.